README.md

```diff
@@ -240,8 +250,62 @@ table.Scan(startedBefore (DateTimeOffset.Now - TimeSpan.FromDays 1.))
 
 (See [`Script.fsx`](src/FSharp.AWS.DynamoDB/Script.fsx) for example timings showing the relative efficiency.)
 
+## `TransactWriteItems`
+
+[`TransactWriteItems`](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TransactWriteItems.html) lets you (at a high level; there are [some key differences](https://stackoverflow.com/a/71706015/11635) in how the APIs are structured)
+compose multiple write transactions into an aggregate request that will succeed or fail atomically. See this [excellent overview article](https://www.alexdebrie.com/posts/dynamodb-transactions) by [@alexdebrie](https://github.com/alexdebrie) and
+[the `TransactWriteItems` API documentation](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TransactWriteItems.html) for full semantics
+(e.g., the full API lets you compose transactions across tables - the present implementation does not attempt to expose that facility,
+and there are diverse constraints such as the fact that no item may have more than one operation applied to it within the overall transaction).
```

Comment:
This feels a bit heavy on explanation of (and external links regarding) the Dynamo feature, rather than how the library supports it. Probably just really needs to be something like "[TransactWriteItems](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TransactWriteItems.html) is supported, but currently only across a single table"

Reply:
Sending a PR with a tighter description - feel free to edit afterward